### ISSUE TITLE
getting error Started getting error: Could not load file or assembly RestSharp, Version=105.0.1.0

### ISSUE THREAD

**Issue by sstasevich** [shawnmclean/Mandrill-dotnet#125]
after upgrade to latest version 1.2.7 that has RestSharpSigned dependency
starting getting error everywhere where restsharp(regular) is used

example
An exception of type 'System.IO.FileLoadException' occurred in Twilio.Api.dll but was not handled in user code

**Comment by webprofusion-chrisc** [comment]
Also seeing this. Fixing it seems to require a fall back to the older version of RestSharp. Is there any way the dependency on RestSharp could be removed?

**Comment by sstasevich** [comment]
That was really annoying, but yes Rolling back 2 versions to one with RestSharp not RestSharpSigned fixed that.

**Comment by webprofusion-chrisc** [comment]
The alternative seems to be to completely remove the Mandrill.net nuget package and take the RestSharp install with it, then reinstall.

**Comment by shawnmclean** [comment]
Do you guys suggest that I create an alternate Mandrill.net nuget package for signed dlls? It seemed this may cause some issues with other libraries using `RestSharp` as a dependency.

**Comment by sstasevich** [comment]
Yes, I think the Mandrill.net Signed as a separate package so people can
choose would be great.

**Comment by PureKrome** [comment]
How about this [epic thread](https://github.com/octokit/octokit.net/issues/405) that has discussed this issue in detail?

TLDR: kill off signing. people that need it signed, let them do it themselves. (yeah, u can provide signed nuget's but IMO it still doesn't really highlight the vote you're taking).

**Comment by roryprimrose** [comment]
Just hit this as well. The issue is actually with the RestSharpSigned assembly. It is compiled to be version 100.0.0.0 which breaks everything that attempts to reference RestSharp 105.X.X.X.
